tests/core/json_configs/test_native_tf2_cprofile_profiler_config_parser.json

```diff
@@ -2,7 +2,7 @@
     "ProfilingParameters": {
         "ProfilerEnabled": true,
         "LocalPath": "/tmp/test",
-        "DetailedProfilingConfig": "{\"StartStep\": 2, \"NumSteps\": 2}",
-        "PythonProfilingConfig": "{\"StartStep\": 5, \"NumSteps\": 2, \"ProfilerName\": \"cProfile\"}"
+        "DetailedProfilingConfig": "{\"StartStep\": 0, \"NumSteps\": 2}",
+        "PythonProfilingConfig": "{\"StartStep\": 0, \"NumSteps\": 2, \"ProfilerName\": \"cProfile\"}"
```

Comment:
Why?

Reply:
Thanks for catching, forgot to revert these changes.